### PR TITLE
improvement: Removed extra white spaces and blank lines.

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Removed extra white spaces and blank lines.
- Removed extra PHP tags.

### Types of changes
- Improvements.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards

### CODE REVIEW

File: inc/assets/css/customizer-preview.css [hunk 1]
```diff
@@ -78,6 +78,19 @@
     text-shadow: none;
 }
 
+.elementor-page .customize-partial-edit-shortcut button {
```
Comment (premanshup) [inline inc/assets/css/customizer-preview.css:81]: @sagarb90 PLease verify why we are adding this CSS?

File: inc/builder/type/footer/copyright/class-astra-footer-copyright-component-loader.php [hunk 2]
```diff
@@ -43,15 +43,6 @@ public function preview_scripts() {
 		$file_prefix = ( SCRIPT_DEBUG ) ? '' : '.min';
 		wp_enqueue_script( 'astra-footer-copyright-customizer-preview-js', ASTRA_BUILDER_FOOTER_COPYRIGHT_URI . '/assets/js/customizer-preview.js', array( 'customize-preview', 'astra-customizer-preview-js' ), ASTRA_THEME_VERSION, true );
 
-		// Localize variables for Astra Breakpoints JS.
```
Comment (premanshup) [inline inc/builder/type/footer/copyright/class-astra-footer-copyright-component-loader.php:46]: @sagarb90 PLease verify why we are removing this JS?

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/assets/css/customizer-preview.css (hunk 1 above)
Comment (sagarb90) [reply inc/assets/css/customizer-preview.css]: @premanshup this is other developer commit changes here we have just worked on HTML blank spaces. Now I have updated with latest branch.

File: inc/builder/type/footer/copyright/class-astra-footer-copyright-component-loader.php (hunk 2 above)
Comment (sagarb90) [reply inc/builder/type/footer/copyright/class-astra-footer-copyright-component-loader.php]: @premanshup Same as above comment.

File: inc/assets/css/customizer-preview.css (hunk 1 above)
Comment (premanshup) [reply inc/assets/css/customizer-preview.css]: @sagarb90 Got it 👍🏼

File: inc/builder/type/footer/copyright/class-astra-footer-copyright-component-loader.php (hunk 2 above)
Comment (premanshup) [reply inc/builder/type/footer/copyright/class-astra-footer-copyright-component-loader.php]: @sagarb90 👍🏼

**Review by shubhw12 — APPROVED** [review]

**Review by premanshup — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**